contents/blog/the-posthog-array-1-41-0.md

```diff
@@ -0,0 +1,165 @@
+
+> **If you have completed async migration 0007 before:** If you have changed anything regarding `alias` or based on ingestion warnings, you must re-run async migration 0007 by [connecting to Postgres](/docs/self-host/deploy/troubleshooting#how-do-i-connect-to-postgres), running `UPDATE posthog_asyncmigration SET status = 0 WHERE name = '0007_persons_and_groups_on_events_backfill' AND status = 2;` and re-running 0007 at `<your-posthog-site>/instance/async_migrations`.
+
+## PostHog 1.41.0 release notes
+
+**Release highlights:**
+
+- New: Persons on events on by default
+- New: Count of events per user
+- New: Text cards on dashboards
+- New: Ingestion warnings
+- New: App metrics
+- New: View recordings from anywhere
+- New: Change your email yourself
+- New: Hedgehog mode
+- Improved: Recordings interface
+- One more thing: Site apps 
+
+### New: Persons on events on by default
+
+We used to store events in one table and persons in another table. That meant that, once you reached billion event scale, any query which touched person properties would time out. But, no longer! After running [an extensive beta since update 1.39.0](/blog/the-posthog-array-1-39-0#beta-improving-query-performance-by-combining-persons-and-events), we've now added person data onto the events themselves. This change also applies to groups, adding group properties and aggregation to the same table. 
+
+You won’t see any UI changes as a result of this change — persons will still have their own Persons & Groups section on the sidebar, for example — but you will notice results are a lot (up to 400%!) faster for any queries involving persons, groups or events. This is a massive change, so be sure to [read the full announcement for more info](/blog/persons-on-events).
+
+> **Note:** As a result of putting persons on events we now have to be strict about identified and unidentified users. Now, only anonymous users can be aliased into other users - meaning that once you call `identify` on a user, you can no longer alias the user into another identified user. Because of that, we recommend double checking your `identify` and `alias` calls, and following advice above [how to upgrade a self-hosted instance to 1.41.0](/blog/the-posthog-array-1-41-0). 
+
+### New: Count of events per user
+
+![count events per user](../images/blog/array/1-41-0-count-events-per-user.png)
+
+Have you ever asked yourself '*What's the average average number of purchases per user?*' or '*What's the maximum number of forms submitted per user?'* 
+
+Questions like these used to be hard to answer with PostHog, but no more! We've added a new series option for insights, enabling you to see the count of events per user - including sub-options for seeing the averages, minimum and maximum count of events per user. 
```

Comment:
I think the most important goal that can now be achieved is _measuring intensity_. Also, if we're listing functions, we can just mention all of them since that's not an unwieldy long list:
```suggestion
Questions like these used to be hard to answer with PostHog, but no more! Use the new "Count per user" aggregation mode available in Trends to analyze _how intensely_ your users use the product and its features. "Count per user" supports common statistical functions for crunching the per-user numbers: average, median, minimum, maximum, 90th/95th/99th percentile.
```